docs/invitation-instructions.md

```diff
@@ -0,0 +1 @@
+docs/invitation_instructions.md
```

Comment:
Oh, it doesn't seem like this will work as intended: https://github.com/Jigsaw-Code/outline-client/blob/1329-invitation-instructionsmd-link-is-broken/docs/invitation-instructions.md

Maybe we need an actual page saying Moved to <link>

Reply:
ah merp